react/src/themes/index.tsx

```diff
@@ -3,15 +3,17 @@ import React, { useContext } from 'react';
 import { Theme } from './Theme';
 import orange from './themes/orange';
 import paybutton from './themes/paybutton';
+import xec from './themes/xec';
 
-export const themes: Record<ThemeName, Theme> = { orange, paybutton };
+export const themes: Record<ThemeName, Theme> = { orange, paybutton, xec };
```

Comment:
lets double up on the theme names - orange, green, blue, btc, bch, xec. 
orange and btc would be the same theme, blue and xec, etc.

Reply:
JK. new plan: 1 name for each theme, and the names will be 
bcha - orange
bch - green
xec - blue

Reply:
nvm that can be its own branch